### PR TITLE
XW-2913 | add NS_FILE pages to local sitemap

### PR DESCRIPTION
**Pull request by rybmat**
https://wikia-inc.atlassian.net/browse/XW-2913

@Wikia/x-wing

### CODE REVIEW

**Review by idradm — APPROVED** [review]

**Review by latata — APPROVED** [review]

File: extensions/wikia/LocalSitemapPage/LocalSitemapSpecialPage.class.php [hunk 1]
```diff
@@ -68,7 +69,13 @@ public function execute( $par ) {
 
 		$from = $request->getVal( 'namefrom', null );
 		$to = $request->getVal( 'nameto', null );
+		$ns = $request->getInt( 'ns', -1 );
 
-		$this->showToplevel( NS_MAIN, $from, $to );
+		if ( $ns == -1 || ( $ns != NS_MAIN && $ns != NS_FILE ) ) {
```
Comment (latata) [inline extensions/wikia/LocalSitemapPage/LocalSitemapSpecialPage.class.php]: Can't we use triple (=== or !==) operators?

File: extensions/wikia/LocalSitemapPage/LocalSitemapSpecialPage.class.php [hunk 2]
```diff
@@ -68,7 +69,13 @@ public function execute( $par ) {
 
 		$from = $request->getVal( 'namefrom', null );
 		$to = $request->getVal( 'nameto', null );
+		$ns = $request->getInt( 'ns', -1 );
```
Comment (idradm) [inline extensions/wikia/LocalSitemapPage/LocalSitemapSpecialPage.class.php]: Why do we provide -1 as default namespace here?

Comment (idradm) [reply extensions/wikia/LocalSitemapPage/LocalSitemapSpecialPage.class.php]: https://github.com/Wikia/app/blob/55eb3957b7916f8c7f33a9f2aa4fc558f35bb763/includes/Defines.php#L50-L50

File: extensions/wikia/LocalSitemapPage/LocalSitemapSpecialPage.class.php (hunk 1 above)
Comment (idradm) [reply extensions/wikia/LocalSitemapPage/LocalSitemapSpecialPage.class.php]: Why do you enforce NS_MAIN and NS_FILE in case of other namespace provided?

**Review by idradm — APPROVED** [review]

**Pull request merged**